core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -377,14 +394,6 @@ var StoryEditorPage = function() {
       closeSaveModalButton,
       'Commit message modal takes too long to disappear.');
     await waitFor.pageToFullyLoad();
-
-    // Wait for the "Save Draft" button to be reset.
-    if (width > 1000) {
```

Comment:
Just curious, why are we removing this?

Reply:
Because, now this is not preserved in this feature.